src/ngraph/opsets/opset3.hpp

```diff
@@ -0,0 +1,30 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/ops.hpp"
+#include "ngraph/opsets/opset2.hpp"
```

Comment:
Why we include this header if later on in `opset3_tbl.hpp` you anyway `#include "opset2_tbl.hpp"` ? Is it necessary?

Reply:
I've followed the same approach as the person who created opset2 but I think you're right - I've removed this include from opset3.hpp